include/deal.II/grid/grid_tools.h

```diff
@@ -1305,6 +1305,107 @@ namespace GridTools
       const bool                                             perform_handshake,
       const bool enforce_unique_mapping = false);
 
+
+    /**
+     * Data structure returned by
+     * GridTools::internal::distributed_compute_intersection_locations(). It can
+     * be converted to
+     * GridTools::internal::DistributedComputePointLocationsInternal, which can
+     * be used to reinit Utilities::MPI::RemotePointEvaluation.
+     */
+    template <int structdim, int dim, int spacedim>
```

Comment:
The `dim` is not needed here but only in the function `convert_to_distributed_compute_point_locations_internal()` below. I think we should move the template argument there.

Reply:
Done :) [bfa035b](https://github.com/dealii/dealii/pull/15165/commits/bfa035b33343d7686af5610847078ae34286b3a4)

Reply:
[bfa035b](https://github.com/dealii/dealii/pull/15165/commits/bfa035b33343d7686af5610847078ae34286b3a4)